modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustClientCodegen.java

```diff
@@ -174,6 +178,10 @@ public RustClientCodegen() {
         typeMapping.put("object", "serde_json::Value");
         typeMapping.put("AnyType", "serde_json::Value");
 
+        unsignedMapping = new HashMap<>();
+        unsignedMapping.put("i32", "u32");
+        unsignedMapping.put("i64", "u64");
```

Comment:
Thoughts on being able to also support other types? (specifically looking at u8, u16, i8, i16). Could use the same bounds checks to try and intelligently select the type

Reply:
Could definitely force things to the smallest possible size.  Went with these initially as they are the only ones defined in the API spec.